src/iuse_actor.cpp

```diff
@@ -1021,6 +1021,47 @@ iuse_actor *deploy_furn_actor::clone() const
     return new deploy_furn_actor( *this );
 }
 
+void deploy_furn_actor::info( const item &, std::vector<iteminfo> &dump ) const
+{
+    std::string message =
```

Comment:
This should probably go into the `if` clause at the end of the function. It's not used before there and the `else` clause of that function assigns a different string anyway.

Reply:
You're right about where I declared the variable; originally I had a plan where that made sense, but obviously things evolved.

I'd originally avoided inlining because of my reflex that it makes the code messy, but I always forget that astyle helps with that immensely.